backend/pom.xml

```diff
@@ -139,7 +139,33 @@
                     </execution>
                 </executions>
             </plugin>
-
+            <plugin>
+                <groupId>cz.habarta.typescript-generator</groupId>
+                <artifactId>typescript-generator-maven-plugin</artifactId>
+                <version>3.2.1263</version>
+                <executions>
+                    <execution>
+                        <id>generate</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <phase>process-classes</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <jsonLibrary>jackson2</jsonLibrary>
+                    <classes>
+                        <class>web.dtos.CertificateDto</class>
```

Comment:
This works, but there is config to import all classes by path instead of manually adding each class.